projects/vs/psa-boot-test/psa-boot-test.cpp

```diff
@@ -5,72 +5,222 @@
-		{
-			int str_len = strlen((char*)str1) > strlen((char*)str2) ? strlen((char*)str2) : strlen((char*)str1);
-
-			return strncmp((char*)str1, (char*)str2, str_len) == 0;
-		}
-
-	public:
-		/// <summary>
-		/// Tests boot_decrypt_upgrade_package() function using a valid
-		/// upgrade package
-		/// </summary>
-		TEST_METHOD(decryption_test_valid)
-		{
-			boot_upgrade_package_t* package = GET_TEST_PACKAGE(td_valid_package);
-			{
-				bool isEqual = strings_are_equal(package->image, td_valid_package_plain_image_content);
-
-				/* We should have encrypted image so, strings should not be the same before decryption */
-				Assert::IsFalse(isEqual);
-				
-				/* Let us decrypt the valid package */
-				boot_decrypt_upgrade_package(package);
-
-				isEqual = strings_are_equal(package->image, td_valid_package_plain_image_content);
-
-				/* After the encryption, the image must be the same with expected plain image */
-				Assert::IsTrue(isEqual);
-			}
-			RELEASE_TEST_PACKAGE(package);
-		}
-	};
+    return memcmp(arr1, arr2, len) == 0;
 }
+
+namespace PSABootTests
+{
+    TEST_CLASS(PositiveCases)
+    {
+    public:
+         /// <summary>
+         /// In this case, we use a valid package 
+         /// that is encrypted with the encryption key. 
+         /// The decrypted image must be the same as 
+         /// the expected plain image value.
+         /// </summary>
+        TEST_METHOD(decryption_test_valid_image)
+        {
+            boot_upgrade_package_t* package = GET_TEST_PACKAGE(td_package_valid);
+            {
+                bool isEqual = arrays_are_equal(package->image, td_valid_package_plain_image_content, td_valid_package_plain_image_len);
+
+                /* We should have encrypted image so, strings should not be the same before decryption */
+                Assert::IsFalse(isEqual);
+
+                /* Let us decrypt the valid package */
+                boot_decrypt_upgrade_package(package);
+
+                isEqual = arrays_are_equal(package->image, td_valid_package_plain_image_content, td_valid_package_plain_image_len);
+
+                /* After the encryption, the image must be the same with expected plain image */
+                Assert::IsTrue(isEqual);
+            }
+            RELEASE_TEST_PACKAGE(package);
+        }
+        /// <summary>
+        /// In this case, we use a valid package 
+        /// that is signed with the signing private key.
+        /// The image must be authenticated 
+        /// using the signing public key. 
+        /// </summary>
+        TEST_METHOD(authenticate_test_valid_image)
+        {
+            boot_upgrade_package_t* package = GET_TEST_PACKAGE(td_package_valid);
+            {
+                /* Let us the authenticate valid package */
+               bool isEqual = boot_authenticate_upgrade_package(package);
+                Assert::IsTrue(isEqual);
+            }
+            RELEASE_TEST_PACKAGE(package);
+
```

Comment:
Remove the empty space

Reply:
Please fix. 
In general, please be consistent for all all source file formatting. If you put empty line in one case, please apply for all, Or; never add empty line.